src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -32,17 +32,33 @@
     private static final CustomItem generatingItem = new CustomItem(Material.ORANGE_STAINED_GLASS_PANE,
         "&cNot Generating..."
     );
-    public static int ADVANCED_DAY_RATE = 80;
-    public static int ADVANCED_NIGHT_RATE = 10;
-    public static int ADVANCED_OUTPUT = 320;
+    public static int BASIC_DAY_RATE = 10;
+    public static int BASIC_NIGHT_RATE = 1;
+    public static int BASIC_OUTPUT = 0;
+    public static int BASIC_STORAGE = 10_000;
+    public static int LV_DAY_RATE = 100;
+    public static int LV_NIGHT_RATE = 10;
+    public static int LV_OUTPUT = 0;
+    public static int LV_STORAGE = 25_000;
+    public static int MV_DAY_RATE = 200;
+    public static int MV_NIGHT_RATE = 25;
+    public static int MV_OUTPUT = 0;
+    public static int MV_STORAGE = 50_000;
+    public static int HV_DAY_RATE = 400;
+    public static int HV_NIGHT_RATE = 50;
+    public static int HV_OUTPUT = 0;
+    public static int HV_STORAGE = 100_000;
+    public static int ADVANCED_DAY_RATE = 800;
+    public static int ADVANCED_NIGHT_RATE = 100;
+    public static int ADVANCED_OUTPUT = 0;
     public static int ADVANCED_STORAGE = 320_000;
-    public static int HYBRID_DAY_RATE = 640;
-    public static int HYBRID_NIGHT_RATE = 80;
-    public static int HYBRID_OUTPUT = 1200;
+    public static int HYBRID_DAY_RATE = 6_400;
+    public static int HYBRID_NIGHT_RATE = 800;
+    public static int HYBRID_OUTPUT = 0;
```

Comment:
They dont output energy anymore

Reply:
thats the one u dont use

Reply:
Whut?

Reply:
Who told you that

Reply:
the OUTPUT isnt used anywhere u only use day rate and night rate

Reply:
If its not used anywhere why dont you remove it